templates/common/infra/bicep/core/host/aks-agent-pool.bicep

```diff
@@ -7,11 +7,11 @@ param name string
 @description('The agent pool configuration')
 param config object
 
-resource aksCluster 'Microsoft.ContainerService/managedClusters@2023-03-02-preview' existing = {
+resource aksCluster 'Microsoft.ContainerService/managedClusters@2023-07-02-preview' existing = {
```

Comment:
Just wondering if we should pick up a non preview version of the API.  I see a bunch of newer versions listed here: https://learn.microsoft.com/en-us/azure/templates/microsoft.containerservice/allversions, not sure if something like 2023-11-01 would have everything we needed.  Maybe not because of the web-routing addon that is in preview?

Reply:
We need the preview bits for some of the preview features like web app routing to work as expected.  Upgraded to the latest working preview version `2023-10-02-preview`.